packages/errors/src/codes.ts

```diff
@@ -59,6 +59,9 @@ export const SOLANA_ERROR__MALFORMED_NUMBER_STRING = 52 as const;
 export const SOLANA_ERROR__TIMESTAMP_OUT_OF_RANGE = 53 as const;
 export const SOLANA_ERROR__RPC_TRANSPORT_HEADER_FORBIDDEN = 54 as const;
 export const SOLANA_ERROR__RPC_TRANSPORT_HTTP_ERROR = 55 as const;
+export const SOLANA_ERROR__RPC_WEBSOCKET_TRANSPORT_CLOSED_BEFORE_MESSAGE_BUFFERED = 56 as const;
+export const SOLANA_ERROR__RPC_WEBSOCKET_TRANSPORT_CONNECTION_CLOSED = 57 as const;
+export const SOLANA_ERROR__RPC_WEBSOCKET_TRANSPORT_FAILED_TO_CONNECT = 58 as const;
```

Comment:
How would you feel about prefixing RPC Subscriptions errors with `RPC_SUBSCRIPTIONS` and following the same convention as `RPC_TRANSPORT_HTTP` — i.e. the concrete name of the transport come after the abstract name of the interface. Something like:

```ts
// Before.
SOLANA_ERROR__RPC_WEBSOCKET_TRANSPORT_CLOSED_BEFORE_MESSAGE_BUFFERED
SOLANA_ERROR__RPC_WEBSOCKET_TRANSPORT_CONNECTION_CLOSED
SOLANA_ERROR__RPC_WEBSOCKET_TRANSPORT_FAILED_TO_CONNECT
SOLANA_ERROR__SUBSCRIPTION_EXPECTED_SERVER_SUBSCRIPTION_ID
SOLANA_ERROR__SUBSCRIPTION_CANNOT_CREATE_SUBSCRIPTION_REQUEST

// After.
SOLANA_ERROR__RPC_SUBSCRIPTIONS_TRANSPORT_WEBSOCKET_CLOSED_BEFORE_MESSAGE_BUFFERED
SOLANA_ERROR__RPC_SUBSCRIPTIONS_TRANSPORT_WEBSOCKET_CONNECTION_CLOSED
SOLANA_ERROR__RPC_SUBSCRIPTIONS_TRANSPORT_WEBSOCKET_FAILED_TO_CONNECT
SOLANA_ERROR__RPC_SUBSCRIPTIONS_EXPECTED_SERVER_SUBSCRIPTION_ID
SOLANA_ERROR__RPC_SUBSCRIPTIONS_CANNOT_CREATE_SUBSCRIPTION_REQUEST
```

I also think the first three should probably live under the subscription-related error codes. Unless I'm missing something.